config/taxonomy/objects/policy_manager_request.yaml

```diff
@@ -0,0 +1,49 @@
+$schema: http://json-schema.org/draft-04/schema#
```

Comment:
No need for both YAMLs and JSON, just keep the JSON if that's what you need.

Reply:
Removed YAMLs.